src/mapping/PetRadialBasisFctMapping.hpp

```diff
@@ -166,6 +155,12 @@ class PetRadialBasisFctMapping : public RadialBasisFctBaseMapping<RADIAL_BASIS_F
   VertexData bgPreallocationMatrixC(const mesh::PtrMesh inMesh);
 
   VertexData bgPreallocationMatrixA(const mesh::PtrMesh inMesh, const mesh::PtrMesh outMesh);
+
+  // loads the initialGuess for a given dimension or allocates the storage for the first iteration
+  void loadInitialGuessForDim(int dimension, int allDimensions, petsc::Vector &destination);
+
+  // stores the initialGuess for a given dimension
+  void storeInitialGuessForDim(int dimension, int allDimensions, petsc::Vector &source);
```

Comment:
I am again confused with the `dimension` vs `dimensions`. Maybe add more details in the comments.

Reply:
Completely rewrote the comments.